packages/react-component-library/src/components/NumberInputE/NumberInputE.test.tsx

```diff
@@ -343,7 +368,7 @@ describe('NumberInputE', () => {
           })
         })
 
-        assertInputValue('1')
```

Comment:
Might be nice to update the describe() name from `and the user types an invalid character` as the entire value is being replaced with `a` rather than an `a` being typed (confused me for a bit :smile: )